methods/coseismic/Coseismic_Requirement_Validation-Ridgecrest_asc.ipynb

```diff
@@ -2,7 +2,9 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.    def is_downloadable(url):
</pre><p>We could remove this small utility. I was going to use it to download my hand-crafted Ridgecrest data, but then I decided it was not necessary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Lets focuss first on getitng the staging ready, we can hand it then back over.</p>